index.js

```diff
@@ -321,8 +339,9 @@ var ModalBox = createReactClass({
       inSwipeArea = false;
       if (this.props.entry === 'top' ? -state.dy > this.props.swipeThreshold : state.dy > this.props.swipeThreshold)
         this.animateClose();
-      else
+      else if (!this.state.isOpen) {
```

Comment:
This check needs to happen to prevent the modal from opening immediately after closing in specific situations.